routes/numbers.js

```diff
@@ -112,8 +117,8 @@ var allTypesRegex = "/:type(date|year|trivia|math)?";
 /* Parses a batch request string into individual numbers.
  * 
  * @param rangesStr: This is the range of numbers (e.g. "1..3,10")
- * @param {*} parseValue: This is the range of numbers (e.g. "1..3,10")
- * @returns number facts
+ * @param {*} parseValue: Accepts a callback function which parses data into a number
```

Comment:
This is unclear. I think you mean getBatchNums Accepts a callback function. parseValue converts data to a number. Can you review this and resubmit.